src/containers/find-offer/offer-card-square/OfferCardSquare.styles.ts

```diff
@@ -13,16 +13,42 @@ export const styles = {
     width: '100%',
     display: 'flex',
     flexDirection: 'column',
-    gap: '8px',
+    gap: '24px',
     justifyContent: 'space-between'
   },
+  cardTopContent: {
```

Comment:
in this and the next section you have a lot of unnecessary styles, CSS should be as simple as posible

Reply:
> in this and the next section you have a lot of unnecessary styles, CSS should be as simple as posible

oh, you are right, removed it